ui/plugins/controller.js

```diff
@@ -329,9 +329,7 @@ treeherder.controller('PluginCtrl', [
             thBuildApi.cancelAllJobs($scope.repoName, rs.revision);
         };
 
-        /**
-         * Test whether or not the selected job is a reftest
-         */
+        // Test to expose the reftest button in the job details navbar
         $scope.isReftest = function() {
```

Comment:
Since this is a copy, I wonder if we could put it in a shared place.  Perhaps add it to ``providers.js`` then it will be available to both the log viewer and the plugins.